client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -86,5 +121,55 @@ def make_ndarray(tensor_proto):
         >>> output = make_ndarray(tensor_proto)
         >>> print(output)
     '''
+    tensor_shape = [d.size for d in tensor_proto.tensor_shape.dim]
+    size = np.prod(tensor_shape, dtype=np.int64)
+    dtype = as_numpy_dtype(tensor_proto.dtype)
```

Comment:
To avoid confusion, please either add tensor_ here and above for:
`tensor_shape, tensor_size/tensor_num_elements, tensor_dtype`
or remove it:
`shape, size/num_elements, dtype`

Reply:
Also I would suggest going withv name `np_dtype` to indicate that this variable holds type in numpy convention (not tf dtype nor basic python type)